packages/flat-i18n/locales/zh-CN.json

```diff
@@ -13,7 +13,7 @@
   "upload-exception": "上传异常",
   "upload-fail": "上传失败",
   "upload-success": "上传成功",
-  "banned": "已禁止学生聊天和举手",
+  "banned": "已禁止学生聊天",
   "unban": "允许学生聊天和举手",
```

Comment:
Does unban text also need to be fixed simultaneously?

Reply:
Oops, you're right. Let me fix that tomorrow.

Reply:
Done.